pandas/tests/indexes/test_multi.py

```diff
@@ -3307,3 +3308,28 @@ def test_duplicate_multiindex_labels(self):
         with pytest.raises(ValueError):
             ind.set_levels([['A', 'B', 'A', 'A', 'B'], [2, 1, 3, -2, 5]],
                            inplace=True)
+
+    def test_use_enum_in_multiindex(self):
```

Comment:
can you use ``@td.skip_if_no('enum')`` here (then do the actual import inside), we have to guard against the tests being run on 2.7 w/o the compat library (by a user).